### DISCUSSION TITLE
Setting up date, time and timezone.

### DISCUSSION THREAD

**Discussion by dvinodbabu** [helloSystem/hello#357] [Ideas]
I am using the hello system os for past 2 months and its been great for most of the part with simplified and stable desktop. 
However as a novice to freebsd iam struggling to setup date,time and timezone. There is no utility as far i know in the desktop to set the date and time. Every time i setup in command line it is reset after system restart.
A utility to set date,time, timezone along with internet time sync is needed.

**Comment by probonopd** [comment]
Hello @dvinodbabu. Yes, this is a known issue. It is not implemented properly yet.
Pending on https://github.com/helloSystem/hello/issues/36.
